src/librustdoc/html/render/print_item.rs

```diff
@@ -578,12 +578,13 @@ fn item_trait(w: &mut Buffer, cx: &Context<'_>, it: &clean::Item, t: &clean::Tra
         info!("Documenting {} on {:?}", name, t.name);
         let item_type = m.type_();
         let id = cx.derive_id(format!("{}.{}", item_type, name));
+        write!(w, "<details class=\"rustdoc-toggle\" open><summary>");
```

Comment:
Can you add a screenshot to show where this change adds a toggle please?

Reply:
Added a before/after screenshot to the PR description.